lib/components/drawer/sidebar.dart

```diff
@@ -212,6 +212,56 @@ class _SidebarState extends State<Sidebar> {
 
       const Divider(),
 
+      //raid
+      Consumer<UserModel>(builder: (context, model, child) {
+        return ListTile(
+          leading: const Icon(Icons.connect_without_contact),
+          title: Text(AppLocalizations.of(context)!.raidAChannel),
```

Comment:
Sidebar only says Raid instead of Raid Channel because then I would have to add another localization if you want me to change that lemme know.